pkgs/applications/video/epgstation/node-packages.nix

```diff
@@ -0,0 +1,10970 @@
+# This file has been generated by node2nix 1.8.0. Do not edit!
```

Comment:
Should this really have its own `node-packages.nix`? I'm always uncomfortable with node packages, 10k lines for one package seems like a *lot*.

Reply:
I made it separate from other node packages because Epgstation needs `devDependencies` to build, which can't be enabled on a per-package basis with node2nix. But looking at the `texlab` package, I think I have a better way to pull in `devDependencies` for Epgstation without affecting other node packages.

Reply:
I've managed to integrate it into `nixpkgs.nodePackages`, so this copy of `node-packages.nix` are now removed.